tensorflow/core/kernels/mkl/mkl_matmul_ops_common.h

```diff
@@ -873,8 +880,13 @@ class MklMatMulPrimitive : public MklPrimitive {
     // Create memory primitive based on dummy data.
     context_.a_mem.reset(
         new dnnl::memory(*context_.a_md, cpu_engine_, DummyData));
+#ifndef DNNL_AARCH64_USE_ACL
```

Comment:
For consistency with the other changes, can you swap this to an `#ifdef` instead of an `#ifndef`?

Reply:
Thanks @cantonios. Fixed in db051a2.